Lib/fontTools/merge.py

```diff
@@ -919,21 +1003,43 @@ def __init__(self, options=None):
 		self.options = options
 
 	def merge(self, fontfiles):
-
-		mega = ttLib.TTFont()
+		# Take first input file sfntVersion
+		sfntVersion = ttLib.TTFont(fontfiles[0]).sfntVersion
+		mega = ttLib.TTFont(sfntVersion=sfntVersion)
 
 		#
 		# Settle on a mega glyph order.
 		#
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+
+		
+		# Check if all fonts have CFF
+		allOTFs = all("CFF " in font for font in fonts)
+
+		cffTables = []
+		if allOTFs:
+			for font in fonts:
+				_desubtoutinize(font)
+				cffTables.append(font['CFF '])
```

Comment:
what if only some of the font contain CFF table. I think we should error here

Reply:
True